rollup.config.mjs

```diff
@@ -11,6 +24,10 @@ export default {
     name: '_satellite'
   },
   plugins: [
+    replace({
```

Comment:
rollup is our production bundler. enusure `REACTOR_KARMA_CI_UNIT_TEST_MODE` is gone.